crates/bevy_render/src/view/window.rs

```diff
@@ -161,7 +160,7 @@ pub fn prepare_windows(
             });
 
         let swap_chain_descriptor = wgpu::SurfaceConfiguration {
-            format: TextureFormat::bevy_default(),
+            format: surface.get_supported_formats(&render_adapter)[0],
```

Comment:
This line is hard to follow in context. IMO we should have a `surface.first_detected_texture` or something that gets called here and has solid docs.

Reply:
IMO the docs on [get_supported_formats](https://docs.rs/wgpu/latest/wgpu/struct.Surface.html#method.get_supported_formats) are good enough.
Handling the case of `get_supported_formats` returning an empty Vec more gracefully would be nice.

Reply:
> Handling the case of get_supported_formats returning an empty Vec more gracefully would be nice.

Done.